e2e/cypress/support/index.js

```diff
@@ -108,11 +111,67 @@ Cypress.on('test:after:run', (test, runnable) => {
     }
 });
 
-// Reset config
 before(() => {
-    cy.apiLogin('sysadmin');
-    cy.apiUpdateConfig();
-    cy.apiInvalidateCache();
+    const admin = getAdminAccount();
+
+    cy.dbGetUser({username: admin.username}).then(({user}) => {
+        if (user.id) {
+            // # Login existing sysadmin
+            cy.apiAdminLogin();
+        } else {
+            // # Create and login a newly created user as sysadmin
+            cy.apiCreateAdmin().then(({sysadmin}) => {
+                cy.apiAdminLogin().then(() => {
+                    cy.apiSaveTutorialStep(sysadmin.id, '999');
+                });
+            });
+        }
+
+        // # Reset config and invalidate cache
+        cy.apiUpdateConfig();
+        cy.apiInvalidateCache();
+
```

Comment:
Not sure if there are tests that uses admin and changes locale. Might be good to reset locale here as well?

Reply:
Done, see https://github.com/mattermost/mattermost-webapp/pull/5723/commits/f981f63118ae1f7fb8c686a28c85d12fc723ae5c